dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1089,12 +1089,12 @@ exception.occurred.while.adding.listener=SRVE8015E: An exception occurred while
 exception.occurred.while.adding.listener.explanation=An exception occurred while trying to add a listener
 exception.occurred.while.adding.listener.useraction=Verify the listener is a valid type and available to the application
 
-error.validate.annotation.extension.class=SRVE8016E: The {0} annotation can not be applied to the class, {1}, because it extends the wrong super class
+error.validate.annotation.extension.class=SRVE8016E: The {0} annotation cannot be applied to the class, {1}, because it extends the wrong super class
```

Comment:
```suggestion
error.validate.annotation.extension.class=SRVE8016E: The {0} annotation cannot be applied to the class, {1}, because it extends the wrong super class.
```